### PR TITLE
 Add several spelling corrections and refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -807,7 +807,7 @@ addeed->added
 addersses->addresses
 addert->assert
 adderted->asserted
-addes->adds, added,
+addes->adds, added, address,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:810]: Also suggest `adders`?
```text
addes->adds, added, adders, address,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1650,6 +1650,7 @@ alocated->allocated
 alocates->allocates
 alocating->allocating
 alocation->allocation
+alocations->allocations
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:1653]: Also suggest `a location`?
```text
alocation->allocation, a location,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -11679,6 +11688,8 @@ distiction->distinction
 distictly->distinctly
 distiguish->distinguish
 distiguished->distinguished
+distination->destination
+distinations->destinations
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:11692]: Also suggest `distinction` ?
```text
distination->destination, distinction,
distinations->destinations, distinctions,

distinative->distinctive
```
https://github.com/search?q=distinative&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -20105,6 +20127,7 @@ loged->logged, lodged, longed,
 loger->logger, lodger, longer,
 loggging->logging
 loggin->login, logging,
+logial->logical
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:20130]:
```text
logially->logically
```
https://github.com/search?q=logially&type=code

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -23613,6 +23636,7 @@ packaing->packaging
 packats->packets
 packe->packed, packet,
 packege->package
+packes->packs, packages,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:23639]: Also suggest `packed` ?
```text
packes->packs, packed, packages,
```

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -24564,11 +24588,18 @@ poistively->positively
...
 poiter->pointer
 poiters->pointers
+poiting->pointing
 poitn->point
+poitnd->pointed
+poitned->pointed
 poitner->pointer
+poitnes->points
 poitning->pointing
+poitns->points
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:24602]:
```text
poitless->pointless
poitlessly->pointlessly
```
* https://github.com/search?q=poitless&type=code
* https://github.com/search?q=poitlessly&type=code

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -30014,6 +30049,8 @@ signifcant->significant
 signifcantly->significantly
 signifficant->significant
 significanly->significantly
+significat->significant
+significatly->significantly
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:30053]:
```text
insignificat->insignificant
insignificatly->insignificantly
```
* https://github.com/search?q=insignificat&type=code
* https://github.com/search?q=insignificatly&type=code

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -33879,6 +33918,7 @@ turtorial->tutorial
 turtorials->tutorials
 Tuscon->Tucson
 tusday->Tuesday
+tuseday->Tuesday
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:33921]:
```text
teusday->Tuesday
tuesdsy->Tuesday
```
* https://github.com/search?q=teusday&type=code
* https://github.com/search?q=tuesdsy&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi added requested mods and rebased. Please re-review, TIA!

**Review by vikivivi — APPROVED** [review]
LGTM

**Pull request merged**